src/utils/update/mapper.utils.ts

```diff
@@ -104,3 +104,20 @@ export const isSameAddress: AddressMatches = (address1: any, address2?: any) =>
     return Object.keys(address1).every(key => address1[key] === address2[key]);
   }
 };
+
+export const lowerCaseAllWordsExceptFirstLetters = (country: string) => {
+  if (!country){
+    return "";
+  }
+
+  const wordsForAllLowerCase = ["AND", "OF", "THE", "DA", "PART"];
+
+  return country.replace(/\w*/g, word => {
```

Comment:
we get some denial of service security warnings for regex expressions so fingers crossed ...

Reply:
Oh no ... from sonarqube?

Reply:
yes, was for Java api but see nothing here so all good.